modules/introduction/partials/new-features-80.adoc

```diff
@@ -120,6 +120,9 @@ Additional information sent by clients at connection time can be found in the lo
 https://jira.issues.couchbase.com/browse/MB-61048[MB-61048]::
 Once faulty remote cluster credentials are fixed, XDCR will now be able to more quickly restart replications that depend on the repaired references.
 
+https://jira.issues.couchbase.com/browse/MB-11575[MB-11575]::
+XDCR now supports the identification of the target cluster in a replication and incoming replications on the target cluster, using the UI and REST API.
```

Comment:
The description of the feature as "identifying the target cluster" is incorrect and confusing.
When you are on a cluster, you can create out-going XDCR clusters where you define the source (current cluster) and the target cluster.  So, when you talk about "target clusters", people are likely to think about target clusters that they are replicating TO from the current cluster.

It should say:
XDCR now supports the identification of Incoming Replications on a cluster.  The Incoming Replications can be seen in the XDCR tab of the UI or using the REST API command `GET /xdcr/sourceClusters` .